examples/generalized_linear_models/GLM-robust.ipynb

```diff
@@ -482,7 +482,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.10"
+   "version": "3.9.7"
```

Comment:
When I look at the RTD preview from the PR, I still see 3.8 in the Watermark section. Is that OK? Will it be updated automatically when merging?

Reply:
I should remove this file from the PR, I opened it by mistake and must have saved when closing out of muscle memory, it has not been edited nor executed and the watermark is right. The metadata version is not a trustworthy source of information on what was used to run the code.